test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4318,6 +4425,41 @@ def cleanup_files(self):
             self.du.rm(path=dyn_files, sudo=True, force=True)
         self.dyn_created_files = []
 
+    def _get_log_type(self, hostname=None):
+        """
+        This function will return which log files to read
+
+        :param hostname: name of host from where to get log messages
+        :return: int or None
```

Comment:
should be int since we are sending a number

Reply:
I think the confusion came up because you haven't mentioned ":type" for 'hostname', so at first glace it seems like the line below is describing the type of 'hostname' argument, when it's actually describing the return type. So, please add a 'type' line for 'hostname', then leave a blank line, and then add the return line